my-react-app/src/app/screens/Login/components/Fields/customInput.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
+
+export const CustomInput = props => {
+  const { label, input, meta } = props;
```

Comment:
You should destructure these in the function declaration, like `CustomInput = ({ label, input, meta }) =>`.

`const { label, input, meta } = props;` is what we use for class components